eggs/src/deps.ts

```diff
@@ -6,4 +6,5 @@ export { existsSync, expandGlobSync, writeJson } from "https://deno.land/std/fs/
 export { bold, green, yellow, red } from "https://deno.land/std/fmt/colors.ts";
 export { assertEquals } from "https://deno.land/std@v0.56.0/testing/asserts.ts";
 export * as base64 from "https://denopkg.com/chiefbiiko/base64/mod.ts";
+export { version } from "https://raw.githubusercontent.com/nestlandofficial/nest.land/master/eggs/src/version.ts";
```

Comment:
We'll want to store this data on the blockchain eventually, but for now, this should be fine.